### PR TITLE
Starting to translate section 3 to PTBR

### PR DESCRIPTION
**Pull request by Lorenzobattistela**
Merged Jmallone:main and my repo to keep it up to date with commit history. Translated lesson 1 of section 3 to PTBR. Please wait all lessons to be translated before merging it! 😃

### CODE REVIEW

**Comment by Lorenzobattistela** [comment]
I've just finished translating section 3 fully.

File: NFT_Collection/pt-br/Section_3/Lesson_1_Setup_React_App_And_Wallet.md [hunk 1]
```diff
@@ -1 +1,31 @@
...
+Eu já criei um projecto básico em react que você pode fazer o  **fork** no Replit.  **Apenas vá [aqui](https://replit.com/@adilanchian/nft-starter-project?v=1), e perto da direita você verá o botão de Fork.** Tenha certeza que você está logado, e clique nisso.
+
+Você irá magicamente clonar meu repositória e a IDE inteira no seu navegador para trabalhar com o código. Uma vez que parar de carregar e mostrar algum código, clique em "run" em cima e você está pronto. Pode levar 2 a 4 minutos na primeira vez.
+
+**Nota: Ao longo desse projeto, você pode perceber que nós estamos referenciando arquivos `.js`. No Replit, se você estiver criando novos arquivos JavaScript, você vai precisar usar a extensão `.jsx`! Replit tem algumas performances chiques que pedem que você usa `.jsx` na extensão de arquivo :).**
+
+🦊 Configurar o Metamask
+---------------------
+
+Incrível, nós temos um projecto em React **já feito o deploy** que podemos trabalhar facilmente. Isso foi simples :).
```
Comment (Jmallone) [inline NFT_Collection/pt-br/Section_3/Lesson_1_Setup_React_App_And_Wallet.md]: **projecto** -> projeto

File: NFT_Collection/pt-br/Section_3/Lesson_1_Setup_React_App_And_Wallet.md [hunk 2]
```diff
@@ -1 +1,31 @@
...
+
+Nota: **Você não precisa usar o replit para construir + fazer deploy do seu site. Se você quer trabalhar localmente no VSCode e usar Vercel/Heroku/AWS para fazer o deploy e está confiante nas suas habilidades com web dev -- está tudo certo. [Aqui](https://github.com/buildspace/buildspace-nft-course-starter) está um link para o repositório base, você pode cloná-lo e trabalhar localmente.**
+
+Crie uma conta no Replit antes de seguir em frente.
+
+Eu já criei um projecto básico em react que você pode fazer o  **fork** no Replit.  **Apenas vá [aqui](https://replit.com/@adilanchian/nft-starter-project?v=1), e perto da direita você verá o botão de Fork.** Tenha certeza que você está logado, e clique nisso.
+
+Você irá magicamente clonar meu repositória e a IDE inteira no seu navegador para trabalhar com o código. Uma vez que parar de carregar e mostrar algum código, clique em "run" em cima e você está pronto. Pode levar 2 a 4 minutos na primeira vez.
```
Comment (Jmallone) [inline NFT_Collection/pt-br/Section_3/Lesson_1_Setup_React_App_And_Wallet.md]: **meu repositória** -> meu repositório

File: NFT_Collection/pt-br/Section_3/Lesson_1_Setup_React_App_And_Wallet.md [hunk 3]
```diff
@@ -1 +1,31 @@
...
+
+Nós vamos usar essa coisa chamada  [Replit](https://replit.com/~)!
+
+É uma IDE no navegador que nos permite construir web apps facilmente e fazer deploy de todos eles pelo navegador. É legítimo. Ao invés de ter que configurar todo um ambiente local e escrever comandos para fazer o deploy, tudo isso é dado para nós.
+
+Nota: **Você não precisa usar o replit para construir + fazer deploy do seu site. Se você quer trabalhar localmente no VSCode e usar Vercel/Heroku/AWS para fazer o deploy e está confiante nas suas habilidades com web dev -- está tudo certo. [Aqui](https://github.com/buildspace/buildspace-nft-course-starter) está um link para o repositório base, você pode cloná-lo e trabalhar localmente.**
+
+Crie uma conta no Replit antes de seguir em frente.
+
+Eu já criei um projecto básico em react que você pode fazer o  **fork** no Replit.  **Apenas vá [aqui](https://replit.com/@adilanchian/nft-starter-project?v=1), e perto da direita você verá o botão de Fork.** Tenha certeza que você está logado, e clique nisso.
```
Comment (Jmallone) [inline NFT_Collection/pt-br/Section_3/Lesson_1_Setup_React_App_And_Wallet.md]: **projecto** -> projeto

File: NFT_Collection/pt-br/Section_3/Lesson_2_Connect_Wallet_To_Web_App.md [hunk 4]
```diff
@@ -1 +1,294 @@
...
+export default App;
+```
+
+## 🔒 Ver se conseguimos acessar a conta do usuário
+
+Então quando você rodar isso, você deve ver aquela linha "We have the Ethereum object" escrita no console do site quando você for inspecioná-lo. Se você estiver usando o Replit, tenha certeza que você está olhando para o console do site do projeto, e não o do Replit! Você pode acessar o console do seu site abrindo ele na sua própria aba e carregando as ferramentas de desenvolvedor. O URL deve se parecer com isso - `https://nft-starter-project.yourUsername.repl.co/`
+
+**BOA.**
+
+Depois, nós precisamos checar se estamos autorizados mesmo a acessar a carteira do usuário. Uma vez que tivermos acesso a isso, podemos chamar noss contrato inteligente.
```
Comment (Jmallone) [inline NFT_Collection/pt-br/Section_3/Lesson_2_Connect_Wallet_To_Web_App.md]: **noss** -> nosso

File: NFT_Collection/pt-br/Section_3/Lesson_3_Create_Button_To_Call_Contract.md [hunk 5]
```diff
@@ -1 +1,147 @@
...
+
+Você deve estar se perguntando o que é gas. Eu não vou responder isso aqui. Mas você pode começar a pesquisar [aqui](https://ethereum.org/en/developers/docs/gas/) ;).
+
+## 🤩 Teste
+
+Você deve estar pronto para mintar uma NFT direto do seu site agora. **Isso é épico!!** Isso é basicamente como todos esses sites de mintar NFT funcionam, e você acabou de fazer você mesmo :).
+
+Eu testei todo o código no vídeo ABI que linkei acima. Tenha certeza de assisti-lo. Eu passo por coisas super importantes sobre o que fazer quando você **muda** o contrato. Porque o seu contrato é permanente, mudanças requerem fazer outro deploy, atualizar o endereço no frontend, e finalmente atualizar o arquivo ABI no frontend
+
+## ✈️ Uma nota em redeploys de contrato
```
Comment (Jmallone) [inline NFT_Collection/pt-br/Section_3/Lesson_3_Create_Button_To_Call_Contract.md]: Será que não fica melhor colocar algo **"Uma nota sobre o redeploy do contrato"** ?

**Review by Jmallone — CHANGES_REQUESTED** [review]
Overall, it's pretty good, 
just make a few changes. 
Congratulations :fire:

File: NFT_Collection/pt-br/Section_3/Lesson_3_Create_Button_To_Call_Contract.md (hunk 5 above)
Comment (Lorenzobattistela) [reply NFT_Collection/pt-br/Section_3/Lesson_3_Create_Button_To_Call_Contract.md]: Acho que sim, fiquei meio em dúvida sobre como colocar mas n tive ideias.

**Comment by Jmallone** [comment]
Looks good!

**Pull request merged**